audbcards/core/dataset.py

```diff
@@ -204,57 +296,58 @@ def license_link(self) -> typing.Optional[str]:
 
         """
         if (
-                self.header.license_url is None
-                or len(self.header.license_url) == 0
+            self.header.license_url is None
+            or len(self.header.license_url) == 0
         ):
             return None
         else:
             return self.header.license_url
 
-    @property
+    @functools.cached_property
     def name(self) -> str:
         r"""Name of dataset."""
         return self.header.name
 
-    @property
+    @functools.cached_property
     def publication_date(self) -> str:
         r"""Date dataset was uploaded to repository."""
-        path = self._backend.join('/', self.name, 'db.yaml')
+        path = self._backend.join("/", self.name, "db.yaml")
         return self._backend.date(path, self._version)
 
-    @property
+    @functools.cached_property
     def publication_owner(self) -> str:
         r"""User who uploaded dataset to repository."""
-        path = self._backend.join('/', self.name, 'db.yaml')
+        path = self._backend.join("/", self.name, "db.yaml")
         return self._backend.owner(path, self._version)
 
     def properties(self):
         """Get list of properties of the object."""
         class_items = self.__class__.__dict__.items()
-        props = dict((k, getattr(self, k))
-                     for k, v in class_items
-                     if isinstance(v, property))
-
+        props = dict(
+            (k, getattr(self, k))
+            for k, v in class_items
+            if isinstance(v, functools.cached_property)
```

Comment:
Here we have the reason why `license` is not passed on as it is a property at the moment and not a `functools.cached_property`. The same holds for `iso_languages`. I guess, you need to change both to be also cached.

Reply:
This indeed fixes issues with license regex and skipped test. Thanks a lot!